test/models/attribute_test.rb

```diff
@@ -217,6 +217,57 @@ def teardown
     )
   end
 
+  test ".store_attribute_value_order_indexes stores ordered value indexes" do
```

Comment:
It feels a little off to me to be testing a private method? Either we just make this public or we test the right public interface that covers this 'side effect'?

Reply:
I'm happy to remove it since its functionality is covered by the test right after it. I was just being thorough